iothub/service/samples/PnpServiceSamples/noComponentSample/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
```

Comment:
The folder should be PnpServiceSamples/Thermostat/Program.cs

Reply:
Sure I can move it to Thermostat if that's what we are going with